scico/numpy/_blockarray.py

```diff
@@ -174,10 +174,18 @@ def prop_ba(self):
 def _da_method_wrapper(method_name):
     method = getattr(Array, method_name)
 
-    if method.__name__ is None:
+    if method.__name__ is None or method.__qualname__ is None:
         return method
```

Comment:
this is no longer needed given the later logic?

Reply:
Good point - fixed.